qemu/tests/cfg/blockdev_inc_backup_merge_external_bitmaps.cfg

```diff
@@ -51,3 +51,7 @@
         remove_image_data1 = no
     iscsi_direct:
         lun_data1 = 1
+    ceph:
+        enable_ceph_base = no
+        enable_ceph_data1sn = no
+        image_format_data1 = raw
```

Comment:
add enable_ceph_inc = no

Reply:
Have defined it in line 43, so no need to add it again in line 58.
  41     # For local target inc image
 42     enable_iscsi_inc = no
 43     enable_ceph_inc = no